src/web/index.css

```
@@ -307,6 +316,17 @@ section {
 	}
 }
 
+@keyframes modalIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "modalIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)